lib/archethic_web/graphql_schema.ex

```diff
@@ -40,6 +40,17 @@ defmodule ArchethicWeb.GraphQLSchema do
       end)
     end
 
+    @desc """
+    Query the network to find the genesis address of a transaction
+    """
+    field :get_genesis_address, :genesis_address do
+      arg(:address, non_null(:address))
+
+      resolve(fn %{address: address}, _ ->
+        Resolver.get_genesis_address(address)
+      end)
+    end
```

Comment:
this part need to be tested, you can have a look at test/archethic_web/graphql_schema_test.exs